app/plugins/view-context.js

```diff
@@ -19,11 +19,13 @@ export const viewContextPlugin = {
           if (path.startsWith("/cookies")) {
             serviceUrl = "/cookies";
           }
+
           ctx.serviceName = serviceName;
           ctx.serviceUrl = serviceUrl;
           ctx.claimServiceUri = claimServiceUri;
           ctx.serviceUri = serviceUri;
           ctx.customerSurveyUri = customerSurvey.uri;
+          ctx.userIsSignedIn = request.auth.isAuthenticated;
```

Comment:
does request.auth always have a value, even for routes where there is no auth? For example when navigating to /cookies does this still work safely?